alf/optimizers/nero_plus.py

```diff
@@ -249,6 +258,8 @@ def step(self, closure=None):
                     sq = grad**2
                 state['exp_avg_sq'].lerp_(sq, 1 - beta2)
                 denom = state['exp_avg_sq'].sqrt().add_(eps)
+                if pweight_decay > 0:
+                    p.mul_(1 - lr_scale * lr)
```

Comment:
missing ```* pweight_decay```?

Reply:
good catch. No wonder the unittest coundn't pass when weight decay is used.